jax/lax/lax_control_flow.py

```diff
@@ -339,7 +344,7 @@ def _cond_batching_rule(args, dims, true_jaxpr, false_jaxpr, true_nconsts,
       args, [1, true_nconsts, true_nops, false_nconsts])
   size, = {x.shape[d] for x, d in zip(args, dims) if d is not batching.not_mapped}
   orig_bat = [d is not batching.not_mapped for d in dims]
-  (pred_bat,), t_bat, tconst_bat, f_bat, fconst_bat = split_list(
+  (pred_bat,), tconst_bat, t_bat, fconst_bat, f_bat = split_list(
```

Comment:
Should this be `true_nops` instead of `len(true_ops)`, following the `split_list(args, ...)` call above?

Reply:
I guess they are the same but using `true_nops` is clearer to me.

Reply:
Ah, you're right, this isn't a bug: I was getting confused with `len(true_jaxpr.in_avals)`. But `true_nops == len(true_ops)` of course!

Either version is good with me; one nice thing about writing `true_nops` is that it matches the unpacking done above, which makes it more obviously correct.